.github/workflows/test-coverage.yml

```
@@ -21,7 +21,9 @@ jobs:
           fetch-depth: 0 # otherwise, there would be errors pushing refs to the destination repository.
       - name: go-test
         run: |
-          go test -v ./... -covermode=count -coverprofile=coverage.out
+          go test -v ./... -covermode=count -coverprofile=coverage.out.temp
+          cat coverage.out.temp | grep -v "/example" > coverage.out
```

Looks good! I think we can remove the `cat` here and just modify the grep to:

```suggestion
          grep -v "/example" coverage.out.temp > coverage.out
```